tests/base_api_tests.py

```diff
@@ -158,19 +158,13 @@ def test_get_filter_related_owners(self):
             API: Test get filter related owners
         """
         self.login(username="admin")
-        argument = {"filter": "a"}
```

Comment:
can we keep this test too? I think it's relevant as it tests multiple different users being returned

Reply:
The endpoint doesn't sort results, so you can't have a deterministic test with multiple results.

Reply:
you could still sort the results in the response and compare? If the order doesn't matter, then we should at least ensure the same users get returned